src/ansys/motorcad/core/geometry.py

```diff
@@ -435,13 +409,37 @@ def edit_point(self, old_coordinates, new_coordinates):
             Position to move the point to
         """
         for entity in self.entities:
+            edited = False
+
             if entity.start == old_coordinates:
                 entity.start = deepcopy(new_coordinates)
+                edited = True
             if entity.end == old_coordinates:
                 entity.end = deepcopy(new_coordinates)
-            if isinstance(entity, Arc):
-                # Check Arc is still valid
-                _ = entity.centre
+                edited = True
+
+            if edited and isinstance(entity, Arc):
+                # Draw line between arc start/end
+                # get centre point of that line
```

Comment:
```suggestion
                # Get center point of that line
```

Reply:
Motor-CAD API methods are British English so trying to have some consistency. Most methods are pre PyAnsys so decided it's too big a change for customers to switch now